examples/example-system.flake.nix

```diff
@@ -79,14 +79,14 @@
                 # expose DNS & the web interface on unpriviledged ports on all IP addresses of the host
                 # check the option descriptions for more information
                 dnsPort = 5335;
-                webProt = 8080;
+                webPort = 8080;
               };
               piholeConfig.ftl = {
```

Comment:
I suppose `piholeConfig.tz` should also get a default, but I wouldn't know how to give it a sensible default/general purpose value/expression